### PR TITLE
feat(admin): add curation review UI for organization evaluations

### LINKED ISSUE #49: Build admin UI for managing organizations
Create admin interface for adding/editing orgs (authenticated users only).
Features:
- Form for creating new orgs
- Edit existing org details
- Upload/link images
- Toggle is_active and is_featured flags
- Preview before publishing
Output: Admin dashboard for org management.

---

**Workstream:** Backend & Data
**Priority:** ⚡ P1
**Sprint:** Sprint 3
**Size:** L
**Item Type:** Feature

**Depends on:**
- #32 Implement basic CRUD API for organizations

### PR DESCRIPTION
**Pull request by lhadjchikh**
## Summary
- Add `OrganizationEvaluation` model to store AI-generated curation evaluations with review status tracking
- Add Django admin interface with approve/reject bulk actions, evidence score display, SDG alignment, and curator notes
- Approved evaluations automatically create `Organization` records from evaluation data
- Reviewer reasoning field for documenting override decisions

## Test plan
- [x] 26 new tests covering model properties, admin list/detail views, approve/reject actions, and edge cases
- [x] All 102 tests pass
- [x] ruff check clean
- [x] 97% code coverage
- [ ] Manual: run `manage.py migrate`, create a superuser, load an evaluation via shell, verify admin UI at `/admin/organizations/organizationevaluation/`

Closes #49

### CODE REVIEW

File: backend/terramedic/organizations/admin.py [hunk 1]
```diff
@@ -16,3 +24,203 @@ class OrganizationAdmin(TranslatableAdmin):
...
+        "evidence_score_display",
+        "recommendation_display",
+        "sdg_display",
+        "status",
+        "reviewer",
+        "reviewed_at",
+    ]
+    list_filter = ["status"]
+    search_fields = ["evaluation_data"]
+    readonly_fields = [
```
[the remark this answers is not in the document]
Comment (lhadjchikh) [reply backend/terramedic/organizations/admin.py:177]: Addressed in bbac6ad — replaced `search_fields` with a `get_search_results` override that queries `evaluation_data__org_metadata__name__icontains`.

File: backend/terramedic/organizations/admin.py [hunk 2]
```diff
@@ -16,3 +24,203 @@ class OrganizationAdmin(TranslatableAdmin):
...
+
+    categories = accessibility.get("categories", [])
+    category = categories[0] if categories else "resource"
+
+    org = Organization(
+        name=meta.get("name", ""),
+        website_url=meta.get("website_url", ""),
+        image_url=meta.get("image_url", ""),
+        category=category,
+        is_active=True,
```
[the remark this answers is not in the document]
Comment (lhadjchikh) [reply backend/terramedic/organizations/admin.py:55]: Addressed in 40d93da — now validates against `Category.values` and defaults to `Category.RESOURCE`.

File: backend/terramedic/organizations/admin.py [hunk 3]
```diff
@@ -16,3 +24,203 @@ class OrganizationAdmin(TranslatableAdmin):
...
+            if note_text:
+                parts.append(f"<p>{note_text}</p>")
+            if flags:
+                parts.append(
+                    "<p><strong>Flags:</strong> "
+                    + ", ".join(flags)
+                    + "</p>",
+                )
+
+        return "\n".join(parts) if parts else "<p>No evaluation data.</p>"
```
[the remark this answers is not in the document]
Comment (lhadjchikh) [reply backend/terramedic/organizations/admin.py]: Addressed in 55ee5da — all user-provided values are now escaped with `django.utils.html.escape()`, then the assembled HTML is marked safe with `mark_safe`.

File: backend/terramedic/organizations/admin.py [hunk 4]
```diff
@@ -16,3 +24,203 @@ class OrganizationAdmin(TranslatableAdmin):
...
+    ]
+    fieldsets = (
+        (
+            "Review",
+            {
+                "fields": (
+                    "status",
+                    "reviewer_reasoning",
+                ),
+            },
```
[the remark this answers is not in the document]
Comment (lhadjchikh) [reply backend/terramedic/organizations/admin.py:198]: Addressed in 3e65bef — `status` is now readonly, forcing reviewers to use the approve/reject bulk actions.

File: backend/terramedic/organizations/admin.py [hunk 5]
```diff
@@ -16,3 +24,203 @@ class OrganizationAdmin(TranslatableAdmin):
...
+                continue
+            org = _create_org_from_evaluation(evaluation)
+            evaluation.organization = org
+            evaluation.status = ReviewStatus.APPROVED
+            evaluation.reviewer = request.user  # type: ignore[assignment]
+            evaluation.reviewed_at = timezone.now()
+            evaluation.save()
+        self.message_user(
+            request,
+            f"Approved {queryset.count()} evaluation(s).",
```
[the remark this answers is not in the document]
Comment (lhadjchikh) [reply backend/terramedic/organizations/admin.py]: Addressed in 4cd4a83 — now reuses `evaluation.organization` when present and reports actual approved count.

File: backend/terramedic/organizations/admin.py [hunk 6]
```diff
@@ -16,3 +24,203 @@ class OrganizationAdmin(TranslatableAdmin):
...
+                continue
+            evaluation.status = ReviewStatus.REJECTED
+            evaluation.reviewer = request.user  # type: ignore[assignment]
+            evaluation.reviewed_at = now
+            evaluation.save()
+        self.message_user(
+            request,
+            f"Rejected {queryset.count()} evaluation(s).",
+            messages.SUCCESS,
+        )
```
[the remark this answers is not in the document]
Comment (lhadjchikh) [reply backend/terramedic/organizations/admin.py:335]: Addressed in 2e1fae8 — reject action now only operates on pending evaluations and reports actual rejected count.

File: backend/terramedic/organizations/admin.py [hunk 7]
```diff
@@ -16,3 +24,203 @@ class OrganizationAdmin(TranslatableAdmin):
...
+        name=meta.get("name", ""),
+        website_url=meta.get("website_url", ""),
+        image_url=meta.get("image_url", ""),
+        category=category,
+        is_active=True,
+    )
+    org.set_current_language("en")
+    org.description = meta.get("description", "")
+    org.action_text = f"Support {meta.get('name', 'this organization')}"
+    org.save()
```
[the remark this answers is not in the document]
Comment (lhadjchikh) [reply backend/terramedic/organizations/admin.py:61]: Addressed in 6ccfefc — `action_text` is now truncated to 100 characters.

File: backend/terramedic/organizations/models.py [hunk 8]
```diff
@@ -40,3 +49,67 @@ class Meta:
...
+        return str(data.get("curator_notes", {}).get("recommendation", ""))
+
+    @property
+    def sdg_numbers(self) -> list[int]:
+        data = self.evaluation_data or {}
+        return [
+            item["sdg"]
+            for item in data.get("sdg_alignment", [])
+            if "sdg" in item
+        ]
```
[the remark this answers is not in the document]
Comment (lhadjchikh) [reply backend/terramedic/organizations/models.py]: Addressed in 58c186b — `sdg_numbers` now coerces values to `int` and skips non-dict items or unconvertible values.
